### PR TITLE
DIG-24, DIG-26, DIG-450, DIG-23, DIG-311

### PR DESCRIPTION
**Pull request by subaha-cob**
DIG-24, DIG-26, DIG-450, DIG-23, DIG-311

### CODE REVIEW

File: docroot/themes/custom/bos_theme/templates/layout/_____page.html.twig [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 <input type="checkbox" id="brg-tr" class="brg-tr" aria-hidden="true" />
 
-<nav class="nv-m">
+<nav class="nv-m" data-test="test">
```
Comment (davidrkupton) [inline docroot/themes/custom/bos_theme/templates/layout/_____page.html.twig:14]: should this be here?

**Review by davidrkupton — APPROVED** [review]
OK LGTM - 1 comment.

**Comment by davidrkupton** [comment]
+https://github.com/CityOfBoston/patterns/pull/700

File: docroot/themes/custom/bos_theme/templates/layout/_____page.html.twig (hunk 1 above)
Comment (subaha-cob) [reply docroot/themes/custom/bos_theme/templates/layout/_____page.html.twig:14]: No. I saw it and took it out. Will be fix in my next PR :)

**Pull request merged**